src/pages/CharacterBuilder.tsx

```
@@ -257,12 +260,57 @@ const CharacterBuilder = (): JSX.Element => {
+      case 3:
+        // Validate all stat fields
+        isValid = await trigger([
+          "stats.agility",
+          "stats.strength",
+          "stats.finesse",
+          "stats.instinct",
+          "stats.presence",
+          "stats.knowledge",
+        ]);
+        // Additionally, ensure all stats have a value other than "none"
+        const allStatsSelected = statKeys.every((key) => formData.stats[key]);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)